docs/how-tos/trigger-sync.md

```diff
@@ -1,61 +1,93 @@
 aliases:
   - /data/trigger-sync/
+  - /services/data/trigger-sync/
 no_service: true
-# SME: Alexa Greenberg
 ---
 
-You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+You may want to sync data only when a certain logic condition is met, instead of at a regular time interval.
 For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
 Or, you may want to trigger sync only when your machine detects an object of a certain color.
 The code for both of these examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/how-tos/create-module/) if you want to use different logic.
 
+To set up conditional syncing you need to:
+
+1. Create or use an existing module that implements a `sensor` component that uses your custom logic to determine when to enable and disable sync.
+   While this sensor is not sensing the time it _senses_ whether the data manager should sync or not.
+2. Change the configuration of the data manager to enable selective sync.
+
 {{% alert title="Note: How sync is triggered" color="note" %}}
 
 Regardless of the specifics of your trigger sync logic, to trigger sync you need to pass `true` to the [CreateShouldSyncReading function](https://pkg.go.dev/go.viam.com/rdk/services/datamanager#CreateShouldSyncReading) within the definition of your modular sensor's `Readings` function.
 See examples in the `Readings` function of the [time-interval-trigger code](https://github.com/viam-labs/trigger-sync-examples-v2/blob/main/time-interval-trigger/selective_sync/selective_sync.go) and the [color-trigger code](https://github.com/viam-labs/trigger-sync-examples-v2/blob/main/color-trigger/selective_sync/selective_sync.go).
 
 {{% /alert %}}
 
+This page covers how to use the `sync-at-time` module.
+You can use this as an example if you use or create a similar module.
+
+{{% alert title="In this page" color="tip" %}}
+
+1. [Add the `sync-at-time` "sensor"](#add-sensor-to-determine-when-to-sync)
+2. [Configure the data manager to sync based on the sensor](#configure-the-data-manager-to-sync-based-on-sensor)
+3. [Test your sync configuration](#test-your-sync-configuration)
+
+{{% /alert %}}
+
 ## Example: `sync-at-time`
 
-This module allows you to configure cloud sync to occur only at a specific time frame by implementing a sensor, `naomi:sync-at-time:timesyncsensor`, that enables sync within a specified time frame and disables sync outside that time frame.
+In this example, you will configure sync to only trigger during a specific time frame of the day using an existing module [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time).
```

Comment:
If you want to move this, this could work well right under the "Add sensor to determine when to sync" heading, before the table